### ISSUE TITLE
Can't install from pip

### ISSUE THREAD

**Issue by blocher** [martsberger/django-sql-utils#1]
This packages is a lifesaver! Thank you.

I am getting this error when installing from pip.

```
Collecting django-sql-utils
  Using cached https://files.pythonhosted.org/packages/f1/ac/8ea2b8202de85c59560e3f7f5f4b57de461c387e512eb40574de11d4fa38/django-sql-utils-0.2.1.tar.gz
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/y4/qbz9b6cx4z5b48vc_y6k_cbr0000gn/T/pip-install-iFRBHZ/django-sql-utils/setup.py", line 13, in <module>
        long_description=read_file('README.rst'),
      File "/private/var/folders/y4/qbz9b6cx4z5b48vc_y6k_cbr0000gn/T/pip-install-iFRBHZ/django-sql-utils/setup.py", line 5, in read_file
        with open(name) as fd:
    IOError: [Errno 2] No such file or directory: 'README.rst'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/y4/qbz9b6cx4z5b48vc_y6k_cbr0000gn/T/pip-install-iFRBHZ/django-sql-utils/
```

I think the README.rst is not uploaded to pip.  Is there anyway to fix this?

**Comment by martsberger** [comment]
Yes, looks like the README is not in the tar archive. I'll update setup.py to be more robust to that file missing.

**Comment by martsberger** [comment]
This should be fixed if you try installing version 0.2.2. @blocher Can you confirm this?

**Comment by blocher** [comment]
Works brilliantly! Thank you.
